geoh5py/objects/drillhole.py

```diff
@@ -733,60 +721,136 @@ def post_processing(self):
                     self._cells = key_map.reshape((-1, 2)).astype("uint32")
                     self.workspace.update_attribute(self, "cells")
 

```

Comment:
This function is doing much more than just validating no?
It's creating new points isin't it?

Reply:
Sure, I can rename.